spinoso-time/src/time/tzrs/offset/error.rs

```diff
@@ -21,12 +21,12 @@ impl fmt::Display for OffsetError {
 }
 
 #[allow(clippy::module_name_repetitions)]
-#[derive(Debug, Clone, PartialEq, Eq)]
-pub struct TzStringError(pub(crate) String);
+#[derive(Debug, Clone, Copy, PartialEq, Eq)]
+pub struct TzStringError;
 
 impl fmt::Display for TzStringError {
     fn fmt(&self, f: &mut fmt::Formatter<'_>) -> Result<(), fmt::Error> {
-        f.write_str(&self.0)
+        write!(f, "expected [+/-]HH[:]MM, A..K, J..Z for utc_offset")
```

Comment:
Ahhh yes, this is where it gets fun. When originally developing this, I was running irb in artichoke - so it was picking up 3.1.2 and I didn't notice until later. I did some double checking, and 2.6.3 only supports the +HH:MM and -HH:MM syntax. It doesn't support the A-IK-Z or +HHMM/-HHMM syntax :facepalm:.

Is it best in this case just to support only the 2.6.3 version?

Reply:
the 2.6.3 target isn't something I take super seriously. The 2.6 line will be out of support soon. I tend to run the latest stable Ruby locally and use that to base MRI behavior off of.

Since we've already got the 3.1.2 behavior implemented, let's keep it.

We should probably rethink what Artichoke's MRI target is and how to message that.

Reply:
Super!

I'll update the message to match Ruby 3 then. Maybe worth while thinking about [this](https://github.com/artichoke/artichoke/pull/1927#discussion_r910298551) as to whether we really want the provided value in the error etc.

Reply:
I've updated to the MRI 3.x value